ecosystem/sep-0024.md

```diff
@@ -150,6 +150,7 @@ Name | Type | Description
 `wallet_name` | string | (optional) In communications / pages about the deposit, anchor should display the wallet name to the user to explain where funds are going.
 `wallet_url` | string | (optional) Anchor should link to this when notifying the user that the transaction has completed.
 `lang` | string | (optional) Defaults to `en`. Language code specified using [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1). `error` fields in the response, as well as the interactive flow UI and any other user-facing strings returned for this transaction should be in this language.
+`claimable_balance_supported` | boolean | (optional) True if submitting transactions as a claimable balance is accepted, false otherwise.
```

Comment:
>`claimable_balance_supported`...a claimable balance is accepted...

Does this field indicate to the server that the client supports claimable balances?

Reply:
That is correct.